### PR TITLE
feat: add support for REPAIR sql stmt

### LINKED ISSUE #658: [SQL] Support MySQL `REPAIR` sql statment.
**What would you like to be added**:

**Why is this needed**:

https://dev.mysql.com/doc/refman/8.0/en/repair-table.html

### PR DESCRIPTION
**Pull request by shawnh2**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #658 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/runtime/ast/repair_table.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+func NewRepairTableStmt() *RepairTableStmt {
+	return &RepairTableStmt{}
+}
+
+func (r *RepairTableStmt) CntParams() int {
+	return 1
+}
+
+func (r *RepairTableStmt) Restore(flag RestoreFlag, sb *strings.Builder, args *[]int) error {
+	sb.WriteString("REPAIR TABLE")
```
Comment (jjeffcaii) [inline pkg/runtime/ast/repair_table.go:43]: I think it should end with a space character.

File: pkg/runtime/ast/repair_table.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	if err := r.Table.Restore(flag, sb, args); err != nil {
+		return errors.Wrapf(err, "an error occurred while restore RepairTableStatement.")
```
Comment (jjeffcaii) [inline pkg/runtime/ast/repair_table.go:45]: Remove the dot character at the end of message.

File: pkg/runtime/plan/ddl/repair_table.go [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+		sb   strings.Builder
+		args []int
+	)
+
+	ctx, span := plan.Tracer.Start(ctx, "RepairTable.ExecIn")
+	defer span.End()
+
+	// currently, only implemented db0
+	db, tb := c.Shards.Smallest()
+	if db == "" {
```
Comment (jjeffcaii) [inline pkg/runtime/plan/ddl/repair_table.go:74]: It's possible that the target table is not a sharding table, you should handle this situation.

**Review by jjeffcaii — COMMENTED** [review]
pls fix

**Review by maronghe — APPROVED** [review]

**Comment by maronghe** [comment]
pls fix github CI/CD bug

File: pkg/runtime/ast/repair_table.go (hunk 1 above)
Comment (shawnh2) [reply pkg/runtime/ast/repair_table.go]: fixed

File: pkg/runtime/ast/repair_table.go (hunk 2 above)
Comment (shawnh2) [reply pkg/runtime/ast/repair_table.go]: fixed

**Review by jjeffcaii — APPROVED** [review]

**Pull request merged**